back/engines/commercial/insights/app/controllers/insights/web_api/v1/classification_tasks_controller.rb

```diff
@@ -48,11 +48,20 @@ def view
           )
         end
 
-        # @return [Array<Idea>, nil]
+        # @return [Idea::ActiveRecord_Relation, nil]
```

Comment:
The underscore looks fishy to me, is it on purpose or a small typo?

```suggestion
        # @return [Idea::ActiveRecordRelation, nil]
```

Reply:
It's on purpose.
```ruby
Idea.all.class
# => Idea::ActiveRecord_Relation
```
It's just rails magic ✨